sass/site2.scss

```diff
@@ -71,41 +71,26 @@ img {
 pre, code {
   white-space: pre;
   word-wrap: normal;
-  overflow-x: auto;
 
   b, strong {
     font-weight: 700;
   }
 }
 
+pre {
+  overflow-x: auto;
+}
+
+
 a code {
   color: currentcolor;
 }
 
 ul, ol {
-  //padding-left: 0;
 
   li::before {
-    content: '·';
     color: $secondary-heading-color;
-    margin-left: -15px;
-    position: absolute;
```

Comment:
Currently we have set list-style to none and are manually formatting with our own bullets, which are really tiny. This just restores standard list styling for bullets.